src/Symfony/Component/Security/Core/Encoder/Argon2iPasswordEncoder.php

```diff
@@ -17,9 +17,23 @@
  * Argon2iPasswordEncoder uses the Argon2i hashing algorithm.
  *
  * @author Zan Baldwin <hello@zanbaldwin.com>
+ * @author Dominik Müller <dominik.mueller@jkweb.ch>
  */
 class Argon2iPasswordEncoder extends BasePasswordEncoder implements SelfSaltingEncoderInterface
 {
+    private $config;
```

Comment:
the default value should be `array()`, otherwise `password_hash($raw, \PASSWORD_ARGON2I, $this->config);` will throw a warning
this should cover @chalasr's comment below

Reply:
Thanks. Done.